src/lib.rs

```diff
@@ -317,67 +317,32 @@ impl<A: Send + 'static> Cancel<A> for Option<A> {
     }
 }
 
-/*
- *
- * ===== Async implementations =====
```

Comment:
Removed because there's an equivalent block above at the `Async` impl for `Result` (that itself would be a bit clearer if the `Cancel` impl for `Option` was moved up a bit to be just below the `Cancel` trait definition, but I didn't want to clutter this diff with unrelated moving of code).